Binary files ser-0.8.14.old/.cfg.lex.swp and ser-0.8.14.dev/.cfg.lex.swp differ
diff -urN ser-0.8.14.old/Makefile.defs ser-0.8.14.dev/Makefile.defs
--- ser-0.8.14.old/Makefile.defs	2004-07-27 13:38:33.000000000 +0200
+++ ser-0.8.14.dev/Makefile.defs	2005-05-01 01:18:12.000000000 +0200
@@ -660,7 +660,7 @@
 endif
 YACC_FLAGS=-d -b cfg
 # on solaris add -lxnet (e.g. LIBS= -lxnet)
-LIBS= -lfl -ldl -lresolv
+LIBS=  -ldl -lresolv
 
 
 #os specific stuff
@@ -695,7 +695,7 @@
 ifeq ($(CC_NAME), suncc)
 	LIBS= -lfast -ldl -lresolv
 endif
-	LIBS+= -L/usr/local/lib -lfl -lxnet -lrt -lnsl 
+	LIBS+= -L/usr/local/lib  -lxnet -lrt -lnsl 
 	# -lrt needed for sched_yield
 endif
 
@@ -705,9 +705,9 @@
 	ifneq ($(found_lock_method), yes)
 		DEFS+= -DUSE_PTHREAD_MUTEX  # try pthread sems
 		found_lock_method=yes
-		LIBS= -pthread -lfl  #dlopen is in libc
+		LIBS= -pthread   #dlopen is in libc
 	else
-		LIBS= -lfl  #dlopen is in libc
+		LIBS=   #dlopen is in libc
 	endif
 	YACC=yacc
 endif
@@ -722,7 +722,7 @@
 	# (symbols on openbsd are prefixed by "_")
 	YACC=yacc
 	# no sched_yield on openbsd unless linking with c_r (not recommended)
-	LIBS= -lfl 
+	LIBS=  
 	OPENBSD_IS_AOUT= $(shell echo "$(OSREL)" | \
 				sed -e 's/^3\.[0-3][^0-9]*$$/yes/' |sed -e 's/^[0-2]\..*/yes/')
 	
@@ -740,7 +740,7 @@
 		found_lock_method=yes
 	endif
 	YACC=yacc
-	LIBS= -lfl 
+	LIBS=  
 endif
 
 # OS X support, same as freebsd
@@ -754,9 +754,9 @@
 	ifneq ($(found_lock_method), yes)
 		DEFS+= -DUSE_PTHREAD_MUTEX  # try pthread sems
 		found_lock_method=yes
-		LIBS= -pthread -lfl -lresolv  #dlopen is in libc
+		LIBS= -pthread  -lresolv  #dlopen is in libc
 	else
-		LIBS= -lfl -lresolv  #dlopen is in libc
+		LIBS=  -lresolv  #dlopen is in libc
 	endif
 	LDFLAGS=        # darwin doesn't like -O2 or -E
 	MOD_LDFLAGS= -bundle -bundle_loader ../../$(MAIN_NAME)
diff -urN ser-0.8.14.old/cfg.lex ser-0.8.14.dev/cfg.lex
--- ser-0.8.14.old/cfg.lex	2004-02-12 17:17:48.000000000 +0100
+++ ser-0.8.14.dev/cfg.lex	2005-05-01 01:23:03.000000000 +0200
@@ -71,7 +71,10 @@
 
 	static char* addstr(char*, char**);
 	static void count();
-
+	int yywrap(void)
+	{
+		return 1;
+	}
 
 %}
 
